--- nant-0.85/src/NAnt.Core/Task.cs.orig	2005-10-21 19:13:06.000000000 +0200
+++ nant-0.85/src/NAnt.Core/Task.cs	2007-05-16 22:13:21.319644703 +0200
@@ -400,11 +400,13 @@
         #region Protected Instance Methods
 
         /// <summary><note>Deprecated (to be deleted).</note></summary>
+#if !MONO
         [Obsolete("Deprecated- Use InitializeTask instead")]
         protected override void InitializeElement(XmlNode elementNode) {
             // Just defer for now so that everything just works
             InitializeTask(elementNode);
         }
+#endif
 
         /// <summary>Initializes the task.</summary>
         protected virtual void InitializeTask(XmlNode taskNode) {
